src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -355,6 +376,13 @@ module.exports = function<T, P, I, TI, C, CX>(config : HostConfig<T, P, I, TI, C
 
   function resetWorkPriority(workInProgress : Fiber) {
     let newPriority = NoWork;
+
+    // Check for pending update priority
```

Comment:
Maybe add a comment that we usually expect this to be `null` so this shouldn't be a perf issue.